packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx

```diff
@@ -79,21 +88,16 @@ Warning.args = {
 
 export const WithLink = DefaultTemplate.bind({});
 WithLink.args = {
-  content: (
-    <>
-      Hangar 18 has been compromised.
-      <Link href="#anchor" variant="monochrome">
-        View report
-      </Link>
-    </>
-  ),
+  content: "Hangar 18 has been compromised.",
+  linkUrl: "#anchor",
+  linkText: "View report",
   role: "status",
   severity: "error",
 };
 
 export const Dismissible = DefaultTemplate.bind({});
 Dismissible.args = {
-  onClose: `{() => {}}`,
+  isDismissable: true,
   severity: "warning",
   role: "status",
 };
```

Comment:
You already removed `onClose`, but so you know, Storybook adds `onX` functions for you and logs them under Actions for you:
<img width="488" alt="image" src="https://user-images.githubusercontent.com/97464104/212149731-49fcf86a-9ff5-40ab-a742-e3a02a1b981c.png">

All though in our Storybook, this may be disabled; I'm not sure. It's not working from my testing. It may need a ticket to get it added back as that's the default behavior.